app/views/partials/_sidemenu.html.erb

```diff
@@ -1,16 +1,26 @@
   <div class="sidenav">
       <div class="sidenav-logo sidenav-item">
-        <i class="fab fa-algolia sidenav-icon"></i>Logo
+        <%= image_tag('AlotLogo/logo.png', alt: "Edit Entry") %>
```

Comment:
remove alt.
there is also AIOT text next to this image can you add that too (without the 'tm' part)